L/Libass/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+name = "libass"
+version = v"0.14.0"
+# Collection of sources required to build LibassBuilder
+sources = [
+    "https://github.com/libass/libass/releases/download/0.14.0/libass-0.14.0.tar.xz" =>
+    "881f2382af48aead75b7a0e02e65d88c5ebd369fe46bc77d9270a94aa8fd38a2",
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+apk add nasm
+cd libass-0.14.0/
+sed -i 's/9.10.3/2.9.1/' configure.ac
```

Comment:
This hack and following autoreconf should not be necessary anymore. Probably also setting AR will be unnecessary

Reply:
Can you directly remove those (github allows to edit files)? I don't really understand these hacks, so the probability that I mess something up while applying your comment is quite high ;)

Reply:
I don't think I can edit the pull request.  My suggestion is to simply delete lines 17 and 18 (these lines were working around an issue with the FreeType pkgconfig file, now fixed with https://github.com/JuliaPackaging/Yggdrasil/commit/d2297a46ccf9e9983028bdee8780e658d48e9f75)

Reply:
Didn't you just suggest an edit? :D

Reply:
Ok, I thought you were suggesting me to push commits to the pull request :sweat_smile: Here you literally only need to delete two lines, it's a bit annoying that you can't suggest multi-line changes.